src/WebJobs.Script/Environment/EnvironmentSettingNames.cs

```diff
@@ -49,6 +49,8 @@ public static class EnvironmentSettingNames
         public const string FunctionsRuntimeScaleMonitoringEnabled = "FUNCTIONS_RUNTIME_SCALE_MONITORING_ENABLED";
         public const string AzureWebsiteStartupContextCache = "WEBSITE_FUNCTIONS_STARTUPCONTEXT_CACHE";
         public const string AzureWebJobsFeatureFlags = "AzureWebJobsFeatureFlags";
+        public const string RoleInstanceId = "RoleInstanceId";
+        public const string HealthPingEnabled = "WEBSITE_FUNCTIONS_HEALTH_PING_ENABLED";
```

Comment:
Including this app setting based disable flag for safety. E.g. if for some particular app/scenario we see scaling problems, we can disable until fixed.